Apps/Database/src/Models/Communication.cs

```diff
@@ -74,5 +75,10 @@ public class Communication : AuditableEntity
         /// The lower the value the lower the priority.
         /// </summary>
         public int Priority { get; set; } = EmailPriority.Standard;
+
+        /// <summary>
+        /// Gets the list containing all the communication emails related to this communication.
+        /// </summary>
+        public virtual List<CommunicationEmail> CommunicationEmails { get; } = null!;
```

Comment:
```= new List<CommunicationEmail>;```

Reply:
this is an old version, not sure why you can still see it there.